### ISSUE TITLE
API documentation should probably be its own section of the developer docs.

### ISSUE THREAD

**Issue by terrycloth** [cozy/cozy-docs#156]
Right now, the Data System, ODM, and Controller APIs are only described in the "Cookbooks" section of the developer docs. A full API reference, which explains the types and semantics of every method and property, etc., could be useful. And these API references should probably be in their own section of the developer documentation.

What do you think?

**Comment by frankrousseau** [comment]
Yes. At some point we'll have a to rethink the whole documentation and make it exhaustive.
